packages/styles/scss/patterns/cardarray/README.md

```diff
@@ -0,0 +1,42 @@
+### SCSS
+
+#### Usage
+
+Import the package css into the top of your main CSS file.
+
+```css
+@import '@carbon/ibmdotcom-styles/scss/patterns/listsection/index';
```

Comment:
This should be referencing `cardarray` and not `listsection`

Reply:
I'm stil working on the readme files, the PR was mainly for the code checking.